backend/src/main/java/com/woowacourse/kkogkkog/member/domain/Member.java

```diff
@@ -38,6 +37,10 @@ public class Member {
     @Column(nullable = false)
     private String imageUrl;
 
+    public Member(Nickname nickname, String email, String imageUrl) {
+        this(null, null, null, nickname, email, imageUrl);
+    }
```

Comment:
👍
이렇게 하고 서비스의 save 로직을 한줄로 묶는건 어떨까요?
```suggestion
    public Member(String nickname, String email, String imageUrl) {
        this(null, null, null, new Nickname(nickname), email, imageUrl);
    }
```

Reply:
변경했습니다